test/sharness/t0100-name.sh

```diff
@@ -10,12 +10,14 @@ test_description="Test ipfs repo operations"
 
 test_init_ipfs
 
+for TTL_PARAMS in "" "--ttl=0s" "--ttl=1h"; do
+
 # test publishing a hash
 
-test_expect_success "'ipfs name publish' succeeds" '
+test_expect_success "'ipfs name publish${TTL_PARAMS:+ $TTL_PARAMS}' succeeds" '
 	PEERID=`ipfs id --format="<id>"` &&
 	test_check_peerid "${PEERID}" &&
-	ipfs name publish "/ipfs/$HASH_WELCOME_DOCS" >publish_out
+	ipfs name publish${TTL_PARAMS:+ $TTL_PARAMS} "/ipfs/$HASH_WELCOME_DOCS" >publish_out
```

Comment:
I used the `:+` expansion in the test title to avoid a string with an extra space like `"'ipfs name publish ' succeeds"` when there is no ttl parameter, and then just used the exact same thing in the actual code. But granted, it’s much more readable that way. How about I keep the `:+` expansion in the title to make the output pretty but use a plain `$TTL_PARAMS` in the code?